doc/whats_new/v1.1.rst

```diff
@@ -610,6 +610,11 @@ Changelog
 - |Fix| Fixed a bug in :func:`metrics.normalized_mutual_info_score` which could return
   unbounded values. :pr:`22635` by :user:`Jérémie du Boisberranger <jeremiedbb>`.
 
+- |Fix| :func:`metrics.ndcg_score` will now trigger a warning when the y_true
+  value contains a negative value. It will allow the user to still use negative
+  values, but the result may not be between 0 and 1.
+  :pr:`22710` by :user:`Conroy Trinh <trinhcon>`
```

Comment:
Try to use \`backticks\` for parameters like `y_true`. This should probably also mention the deprecation of negative values for `y_true`. I made a minor wording edit as well, but feel free to rephrase as you see fit :)

```suggestion
- |Fix| :func:`metrics.ndcg_score` will now trigger a warning when `y_true`
  contains a negative value. The user may still use negative
  values, but the result may not be between 0 and 1. Begins deprecation
  of negative values in `y_true`.
  :pr:`22710` by :user:`Conroy Trinh <trinhcon>`
```